server/weaverbird/backends/pandas_executor/steps/join.py

```diff
@@ -0,0 +1,33 @@
+from pandas import DataFrame, merge
+
+from weaverbird.backends.pandas_executor.types import DomainRetriever, PipelineExecutor
+from weaverbird.pipeline.steps import JoinStep
+
+from .utils.cleaning import rename_duplicated_columns
+from .utils.combination import resolve_pipeline_for_combination
```

Comment:
Should we use relative imports or always from weaverbird? (Idk if there is a good practice for this, ; I think my IDE puts always path from the root module I think when refactoring)

Reply:
Both are fine to me, I just avoid to use relative imports for parents modules (`from ...foo.bar`) because I find it difficult to understand.